x-pack/plugins/infra/public/pages/metrics/settings/source_configuration_settings.tsx

```diff
@@ -25,18 +25,23 @@ import { IndicesConfigurationPanel } from './indices_configuration_panel';
 import { MLConfigurationPanel } from './ml_configuration_panel';
 import { NameConfigurationPanel } from './name_configuration_panel';
 import { useSourceConfigurationFormState } from './source_configuration_form_state';
+import { useMetricsBreadcrumbs } from '../../../hooks/use_metrics_breadcrumbs';
+import { settingsTitle } from '../../../../public/translations';
```

Comment:
```suggestion
import { settingsTitle } from '../../../translations';
```

Reply:
Do you suggest to use `import { settingsTitle } from '../../../translations';`? Because this doesn't work. Unless you suggest moving translations outside public folder. Can you clarify?

Reply:
I was wondering why we leave `public` and then re-enter it in the import path.

Reply:
Ok I see, I must have copy pasted it. It is fixed now, good catch!

Reply:
Sometimes the TypeScript language server's auto-import feature comes up with weird paths :shrug: